src/platform/monitoring/DowntimeNotification/tests/DowntimeNotification.unit.spec.jsx

```diff
@@ -13,9 +15,9 @@ const endTime = moment().add(6, 'hour').toDate();
 let old;
 
 const innerText = 'This is the inner text';
-function getComponent(dependencies = [], getScheduledDowntime = () => {}) {
+function getComponent(dependencies = [], getScheduledDowntime = () => {}, otherProps = {}) {
   return enzyme.shallow(
-    <DowntimeNotification dependencies={dependencies} scheduledDowntime={null} getScheduledDowntime={getScheduledDowntime}>
+    <DowntimeNotification dependencies={dependencies} getScheduledDowntime={getScheduledDowntime} {...otherProps} shouldSendRequest>
```

Comment:
Just a personal preference but indented props are easier to read and change. Github cuts this component's props off. 
![screen shot 2018-05-21 at 16 52 21](https://user-images.githubusercontent.com/4998130/40333760-60415712-5d17-11e8-9b81-4e83cac08129.png)

Reply:
I enjoy a nice horizontal scroll ➡️